src/js/vre/chapter31/config/form.js

```diff
@@ -370,14 +359,14 @@ const formConfig = {
       title: 'Document Upload',
       reviewTitle: 'Documents',
       pages: {
-        dd214Upload: {
+        dischargeDocumentUpload: {
           path: 'documents/discharge',
           title: 'Discharge document upload',
           reviewTitle: 'Discharge document review',
           depends: form => !form.verified,
           uiSchema: {
             'ui:description': DD214Description,
-            dd214: fileUploadUI('Upload your discharge document', {
+            dischargeDocuments: fileUploadUI('Upload your discharge document', {
               endpoint: '/v0/vic/supporting_documentation_attachments',
```

Comment:
Can we change this endpoint? I know it's not going to work, but I don't want us to forget about it and leave it pointed to vic.